modules/configuring-the-operator-logger.adoc

```diff
@@ -0,0 +1,83 @@
+:_module-type: PROCEDURE
+
+[id='configuring-the-operator-logger_{context}']
+= Configuring the {productname-short} Operator logger
+
+[role='_abstract']
+You can change the log level for {productname-short} Operator components by setting the `.spec.devFlags.logmode` flag for the *DSC Initialization*/`DSCI` custom resource during runtime. If you do not set a `logmode` value, the logger uses the INFO log level by default.
+
+The log level that you set with `.spec.devFlags.logmode` applies to all components, not just those in a *Managed* state.
+
+The following table shows the available log levels.
```

Comment:
Would a colon be more appropriate here when introducing the table?

Reply:
Good shout, updated.